data/events/2017-galway.yml

```diff
@@ -10,11 +10,11 @@ startdate:  2017-11-13
 enddate:  2017-11-14
 
 # Leave CFP dates blank if you don't know yet, or set all three at once.
-cfp_date_start:  # start accepting talk proposals.
-cfp_date_end:  # close your call for proposals.
-cfp_date_announce:  # inform proposers of status
+cfp_date_start:  2017-08-12
+cfp_date_end:  2017-09-10
```

Comment:
I suggest extending this to at least the middle of September and preferably use a Friday (15th or 22nd or 29th) as the last day, so people would have the weekend for reviews.